src/UglyToad.PdfPig/PdfFonts/IFont.cs

```diff
@@ -17,6 +18,8 @@ internal interface IFont
 
         CharacterBoundingBox GetBoundingBox(int characterCode);
 
+        bool TryGetPath(int characterCode, out List<PdfSubpath> path);
```

Comment:
Unless mutability provides large benefits I'd suggest keeping this consistent with the existing API and using `IReadOnlyList<T>` where possible, just for consistency.